config/v1/feature_gates.go

```diff
@@ -595,4 +595,11 @@ var (
 						productScope(ocpSpecific).
 						enableIn(DevPreviewNoUpgrade, TechPreviewNoUpgrade).
 						mustRegister()
+
+	FeatureGateIngressControllerLBSubnetsAWS = newFeatureGate("IngressControllerLBSubnetsAWS").
+							reportProblemsToJiraComponent("Routing").
+							contactPerson("gspence").
+							productScope(ocpSpecific).
+							enableIn(TechPreviewNoUpgrade).
```

Comment:
This will now need `DevPreviewNoUpgrade` as well
```suggestion
							enableIn(DevPreviewNoUpgrade, TechPreviewNoUpgrade).
```

Reply:
Done.